msteams-platform/apps-in-teams-meetings/teams-live-share-capabilities.md

```diff
@@ -603,6 +602,9 @@ function onSelectPresentMode(documentId: string) {
 
 Listen to your customers to understand their scenarios before implementing role verification into your app, particularly for the **Organizer** role. There's no guarantee that a meeting organizer be present in the meeting. As a general rule of thumb, all users will be either **Organizer** or **Presenter** when collaborating within an organization. If a user is an **Attendee**, it's usually an intentional decision on behalf of a meeting organizer.
 
+> [!NOTE]
```

Comment:
Can we place this somewhere more prominent? In other docs these sort of limitations are usually placed on the home page of the feature immediately after the introduction e.g. https://learn.microsoft.com/en-us/microsoftteams/platform/apps-in-teams-meetings/teams-apps-in-meetings

Reply:
Right, we've received feedback that a lot of these prominent warnings distract from the core concepts. Let's leave it for now until we hear a critical mass of folks suggesting this is important to them.